Gemfile

```diff
@@ -18,6 +18,8 @@ group :development, :test do
   gem 'webdrivers'
 end
 
```

Comment:
![image](https://github.com/shaaibu7/Blog-app/assets/93039359/20da68ac-424e-4b7a-b19a-0c425b13653b)

- [ ] It would be nice if you fix all the links highlighted in the image above ⬆️ because right now they are broken 😨 Kindly fix them or if you don't have any section for these links then kindly remove them to make your readme descriptive and professional 👍 😊